python/google/protobuf/internal/message_test.py

```diff
@@ -1638,6 +1638,12 @@ def testMapByteSize(self):
     msg.map_int32_foreign_message[19].c = 128
     self.assertEqual(msg.ByteSize(), size + 1)
 
+  def testMapFieldConstruction(self):
```

Comment:
Put this test under testMapConstruction, and need to add tests for both message map and scalar map
https://github.com/google/protobuf/blob/ed4321d1cb33199984118d801956822842771e7e/python/google/protobuf/internal/message_test.py#L1963

Reply:
Done